Refine mgmt-guidance merge messages for service teams (#39198)
Azure/azure-sdk-for-js: .github/workflows/mgmt-guidance.md

Post this comment and stop:
```markdown
## Next Steps to Merge
✅ All CI checks have passed — this PR is ready for review!

> A SDK reviewer will take a look shortly. Once it's approved, you can go ahead and merge it. To make things easier, feel free to [enable auto-merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request) so it merges on its own right after approval.

Author: Good catch — fixed in fafdb12. Changed to "An SDK reviewer". Thanks!

> An SDK reviewer will take a look shortly. Once it's approved, you can go ahead and merge it. To make things easier, feel free to [enable auto-merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request) so it merges on its own right after approval.

Reviewer: ```suggestion
The SDK engineer will review this PR shortly. Once it's approved, you can go ahead and merge it. To make things easier, feel free to [enable auto-merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request) so it gets auto merged right after approval.
```

Author: Applied in 720df1c. Thanks!
